examples/browserChannelClient/browserChannelClient.js

```diff
@@ -21,10 +21,12 @@ var ws = require("ws");
 
 var socket = new ws("ws://localhost:8081/browserChannel"); // eslint-disable-line new-cap
 
+var changeSetting = false;
+
 // When the connection is done, the client tells to the flow manager its id
 
 socket.on("open", function () {
-    console.log("## Socket connected");
+    console.log("## browserChannelClient: Socket connected");
```

Comment:
Is it common for GPII to use `console.log` instead of `fluid.log`?

Reply:
:-) GPII, in general, uses `fluid.log()`.  But, some examples and other demo code do not.  This particular example doesn't require infusion, or kettle for that matter, and only needs the `ws` WebSocket package.

I'm inclined to re-write it as an infusion component and possibly use kettle for the requests.  That way, it would resemble how UIO+ or Morphic clients access the `/browserchannel` endpoint.  But, I'm not sure that's a high priority.

Reply:
@klown, yes I'm not sure either. I'll defer to @amb26 for that.

Reply:
Leaving this with console.log is fine